package.json

```diff
@@ -90,9 +87,6 @@
 		"moodle-client": "^0.5.2",
 		"nanoid": "^2.0.3",
 		"oauth-1.0a": "^2.2.6",
-		"passport": "^0.4.0",
```

Comment:
please verify it is no more in use, found it probably in src/sockets/index.js:15

Reply:
the code in question has not worked since the feathers4 update, is not used, and will be removed anyway: https://ticketsystem.schul-cloud.org/browse/SC-3334

Reply:
I think, the "socket" part that is use it should not in use at the moment. But it exist a very old not merged client branch that use the socket part.